tests/test_core.py

```diff
@@ -289,6 +294,25 @@ def match(self, key, value):
     assert data.get('024') == 'valid value'
 
 
+def test_none_element():
+    """Test none value for a list element."""
+    overdo = dojson.Overdo()
+
+    @overdo.over('b', 'b')
+    @for_each_value
+    @ignore_list_element
+    def match(self, key, value):
+        if value == 2:
+            return None
```

Comment:
I'm using the `ignore_list_element` decorator, that captures the None values, just as the `ignore_value` one does.